hub/api/pkg/sync/sync.go

```diff
@@ -170,7 +170,7 @@ func (s *SyncService) updateResources(job model.SyncJob, repo git.Repo, res []gi
 			for _, t := range v.Tags {
 				tag := model.Tag{Name: t, CategoryID: others.ID}
 
-				txn.Model(&model.Tag{}).Where(&model.Tag{Name: t}).FirstOrCreate(&tag)
+				txn.Model(&model.Tag{}).Where(&model.Tag{Name: strings.TrimSpace(t)}).FirstOrCreate(&tag)
```

Comment:
this doesn't still address the actual cause, does it? the v.Tags contain spaces. I think the Trim must be applied to the parsing logic.